proto/src/determined/trial/v1/trial.proto

```diff
@@ -7,9 +7,37 @@ import "google/protobuf/struct.proto";
 import "determined/experiment/v1/experiment.proto";
 import "determined/checkpoint/v1/checkpoint.proto";
 import "google/protobuf/timestamp.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+
+message Validation {
```

Comment:
I dont think this is quite a "validation" because it only has a searcher metric, not all the metrics associated with a validation. 

I'm unsure if it should be named `Validation` if it's going to stay as is. Thoughts?